### DISCUSSION TITLE
Welcome to sshuttle Discussions!

### DISCUSSION THREAD

**Discussion by brianmay** [sshuttle/sshuttle#566] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Universal-Ramen** [comment]
hi, extracting source code and all that stuff is way over my head, it there a zip file with an executable so i dont have to go through all that stuff?

**Reply by Universal-Ramen** [reply]
also i dont really know how github works so excuse me if im being stupid and there already is a zip file lol

**Comment by brianmay** [comment]
The latest release - assuming this is what you want - is available here: https://github.com/sshuttle/sshuttle/releases/tag/v1.1.1
